sycl/test/union_param/union_kernel_param.cpp

```diff
@@ -0,0 +1,51 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// TODO: Uncomment once test is fixed on GPU
```

Comment:
What is the problem with GPU?

Reply:
There is no problem with GPU. Copy paste issue from  another test syntax. Sorry about this.